README.md

```diff
@@ -148,18 +133,38 @@ Initialize submodules recursively:
 git submodule update --init --recursive
 ```
 
-### 3. Prepare OAuth credentials
+##### 3. Prepare OAuth credentials
 
 See [the documentation](docs/credentials.md).
 
-### 4. Build the project
+##### 4. Build the project
 
 - Open `FluentHub.sln`.
 - Hit 'Set as Startup item' on `FluentHub.Uwp` in the Solution Explorer
 - Build with `DEBUG|x64|FluentHub.Uwp (Universal Windows)`
 
-## License
+## 💳 Credit
+
+- Many thanks to Joseph Beattie ([@josephbeattie](https://github.com/josephbeattie)) for creating our current logo.
+
+## 📱 Contact
+The easiest way to contact us is to join [our Discord](https://discord.gg/8KtRkjq2Q4).
+
+If you would like to ask a question, please reach out to us via Twitter:
+
+- Tomoyuki Terashita, Lead Developer: [@onein528](https://twitter.com/onein528)
+- Jupiter, Developer: [@DeveloperWOW64](https://twitter.com/DeveloperWOW64)
+- Gabriel Fontán, Developer: [@BobbyESPGabiles](https://twitter.com/BobbyESPGabiles)
+- Luandersonn Airton, Developer: [@luandersonn](https://twitter.com/luandersonn)
+
+## ⚖️ License
 
 Copyright (c) 2022 FluentHub Team
 
-Licensed under the MIT license stated in [LICENSE](LICENSE).
+Licensed under the MIT license stated in the [LICENSE](LICENSE).
+
+---
+
+<a href="https://github.com/FluentHub/FluentHub/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=FluentHub/FluentHub" />
+</a>
```

Comment:
Please do not change like that. Some of those are good. but others is literally bad. Can I revert bad one?

Reply:
What's wrong?

Reply:
@onein528

Reply:
You want it in a different position?

Reply:
> Please do not change like that. Some of those are good. but others is literally bad. Can I revert bad one?

Sure! @onein528

Reply:
Ah @onein528 You want me to?

Reply:
No, I will do that.

Reply:
@onein528 Anything else you want me to change?